packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -62,6 +62,14 @@ $ms-Persona-presenceSizeMd: 12px;
 $ms-Persona-presenceSizeLg: 20px;
 $ms-Persona-presenceSizeXl: 28px;
 
+:global(.ms-PickerPersona-container.is-selected) {
```

Comment:
All of the changes in this css file should likely people in the people picker css file.